src/common/server/NetRemoteServer.cxx

```diff
@@ -34,7 +35,7 @@ void NetRemoteServer::Run()
     builder.RegisterService(&m_service);
 
     m_server = builder.BuildAndStart();
-    std::cout << std::format("Started listening on {}", m_serverAddress) << std::endl;
+    LOG_INFO << std::format("netremote server started listening on {}", m_serverAddress) << std::endl;
```

Comment:
I don't think we need `std::endl` here anymore for plog

Reply:
Removed.